src/include/execution/executors/hash_join_executor.h

```diff
@@ -96,17 +96,16 @@ class HashJoinExecutor : public AbstractExecutor {
    * @param right the right child, used by convention to probe the hash table
    */
   HashJoinExecutor(ExecutorContext *exec_ctx, const HashJoinPlanNode *plan, std::unique_ptr<AbstractExecutor> &&left,
-                   std::unique_ptr<AbstractExecutor> &&right)
-      : AbstractExecutor(exec_ctx) {}
+                   std::unique_ptr<AbstractExecutor> &&right);
 
   /** @return the JHT in use. Do not modify this function, otherwise you will get a zero. */
   // Uncomment me! const HT *GetJHT() const { return &jht_; }
 
-  const Schema *GetOutputSchema() override { return plan_->OutputSchema(); }
```

Comment:
It is okay to leave this method in the header file because it's not something the students will have to implement.